backend/utils/error_handlers.py

```
@@ -0,0 +1,268 @@
+Provides consistent error responses and logging without information disclosure.
+"""
+
+import logging
+import traceback
+from functools import wraps
+
+from flask import Flask, jsonify, request, has_request_context
+from flask_jwt_extended.exceptions import JWTExtendedException
+from marshmallow import ValidationError as MarshmallowValidationError
+from mongoengine.errors import DoesNotExist, NotUniqueError, ValidationError
+from werkzeug.exceptions import HTTPException
```

Ruff (0.12.2)

`werkzeug.exceptions.HTTPException` imported but unused

Remove unused import: `werkzeug.exceptions.HTTPException`

(F401)